src/crimson/os/alienstore/CMakeLists.txt

```diff
@@ -38,7 +38,8 @@ endif()
 
 add_library(alien::cflags INTERFACE IMPORTED)
 set_target_properties(alien::cflags PROPERTIES
-  INTERFACE_INCLUDE_DIRECTORIES $<TARGET_PROPERTY:Seastar::seastar,INTERFACE_INCLUDE_DIRECTORIES>)
+  INTERFACE_COMPILE_OPTIONS $<$<COMPILE_LANGUAGE:CXX>:-Wno-non-virtual-dtor>
```

Comment:
the title of this commit reads:

> Use alien interface instead of direct seastar

but it fails to explain why shall we have this change.